addons-tutorial.md

```diff
@@ -0,0 +1,91 @@
+Addons are used to customize the images used for the containers. An addon has *hooks* that are
+invoked at various points in the lifecycle of a container. A good example is an addon which
+enables the SSH service inside a container to provide access from the outside for any kind
+of automation of the container.  
+
+Addons are created independently, and can be attached to individual applications.
+
+<h2 id='heading--build-your-own-addon'>Build your first addon</h2>
+
+Any addon must have a `manifest.yaml` file and at least one hook in the `hooks` directory, e.g:
+
+```text
+manifest.yaml
+hooks/pre-start
+```
```

Comment:
Not needed here - we have it in the reference already, and the user will see it more clearly after creating the files.